requirements/azure.txt

```diff
@@ -0,0 +1,3 @@
+adlfs==2021.8.2
```

Comment:
We also need to include these files in the source distributions. Please add the following contents in the [`./MANIFEST.in`](https://github.com/iterative/dvc/blob/378486dbf53271ec894b4380f60ec02dc3351516/MANIFEST.in):

```
recursive-include requirements *.txt
```